examples/ota-provider-app/esp32/main/include/ota-provider-common/BdxOtaSender.h

```diff
@@ -109,4 +119,7 @@ class BdxOtaSender : public chip::bdx::Responder
     chip::Callback::Callback<OnBdxBlockQuery> * mOnBlockQueryCallback             = nullptr;
     chip::Callback::Callback<OnBdxTransferComplete> * mOnTransferCompleteCallback = nullptr;
     chip::Callback::Callback<OnBdxTransferFailed> * mOnTransferFailedCallback     = nullptr;
+
+    // Null-terminated string representing file designator
+    char mFileDesignator[30];
```

Comment:
I'm assuming you don't want to use `chip::bdx::kMaxFileDesignatorLen` because even that is too big? If so, suggest defining a local value here to be used.

Reply:
Yes, 0xff is too big and `spiffs` has some limitations on the length of file name.